### PR TITLE
EnergyMockup is_ready

### PR DESCRIPTION
**Pull request by marcus-oscarsson**
Hi

 - Small improvement of EnergyMockup is_ready

Marcus

### CODE REVIEW

File: HardwareObjects/abstract/AbstractEnergy.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def __init__(self, name):
         self.getCurrentWavelength = self.get_current_wavelength
 
     def is_ready(self):
-        return True
+        return self._moving
```
Comment (rhfogh) [inline HardwareObjects/abstract/AbstractEnergy.py]: Should this not be:
    return not self._moving

**Comment by rhfogh** [comment]
Not sure I agree with this:

Strategically: Why do we want text and message functions in the HardwareObject? Is this not a UI issue - so that you should get the state from the HO, and then set the text (or icon, or color change, ...) you want for that state at the UI level? Could that text ever be used for anything other than displaying? Or, alternatively, is there not a state, Open/Closed, where you could use the state name for this purpose?

Technically: If we do decide to do this, do we really need separate functions to return the value of a static class attribute? Would it not be better to just use self.OPEN_TEXT and self.CLOSE_TEXT ? Or a readOnly property, if you want to guard against overwriting?

**Comment by marcus-oscarsson** [comment]
@rhfogh Yes, I agree, I maybe submitted this a bit to fast, there are probably a better way of doing this.

File: HardwareObjects/abstract/AbstractEnergy.py (hunk 1 above)
Comment (marcus-oscarsson) [reply HardwareObjects/abstract/AbstractEnergy.py]: Fixed :)

**Comment by marcus-oscarsson** [comment]
Thanks for the comments, I fixed the is_moving and removed the commits related to the action labels

**Pull request merged**